mediagrains/gsf.py

```diff
@@ -86,8 +94,11 @@ def load(fp, cls=None, parse_grain=None, **kwargs):
     wish to use a custom Grain constructor pass it as parse_grain. The
     defaults are GSFDecoder and Grain. Extra kwargs will be passed to the
     decoder constructor."""
-    s = fp.read()
-    return loads(s, cls=cls, parse_grain=parse_grain, **kwargs)
+    if cls is None:
+        cls = GSFDecoder
+    if parse_grain is None:
+        parse_grain = Grain
```

Comment:
Unused? I didn't see it in subsequent PRs either.

Reply:
Argh, good catch, will fix.